doc/mloginfo.rst

```diff
@@ -393,11 +396,106 @@ For example:
    invoice-prod.invoices     insert       12768411     22233323        86313                12344
    invoice-prod.invoices     insert       12868411     22233323        86313                12344
 
+Sharding (``--sharding``)
+-----------------------------------------
+
+Outputs information of sharding configuration. Includes groupings of
+similar error/warning messages that are specific to sharding. A breakdown
+of chunk migration activity is included, specifying the source/destination
+shard, the number of chunk migrations attempted within the hour, and the
+ratio of successful to failed migrations. Failed chunk migrations specify
+when the failure occurred in formatted as [hours:minutes:seconds:milliseconds]
```

Comment:
The "Failed chunk migrations ..." sentence needs some rephrasing for clarity. Since you are going to show an example below you could trim some of the details like {hours:minutes:..) formatting.

I'm also unsure what "precedes to be successful" means.

Reply:
I removed the sentences that wasn't necessary